internal/tjson/tjson.go

```diff
@@ -177,7 +177,7 @@ func Unmarshal(data []byte, schema *Schema) (any, error) {
 			fallthrough
 		case Int64, Int32:
 			fallthrough
-		case Byte, UUID, DateTime:
+		case Byte, UUID, DateTime, Regex:
```

Comment:
I guess, we don't need `Regex` here

Reply:
Yep I know it even cannot compile when it's called here :)

Reply:
Sometimes I push code and run `task all` in parallel to save a few minutes and after checks are passed on CI I rerequest for review :) Sorry for that